db/schema.prisma

```diff
@@ -51,3 +53,18 @@ model LeaveRequest {
 
   @@unique([id, clubId, startDate, endDate]) // Same user's request for a club can't overlap in time
 }
+
+model ClubRating {
+  id        String        @id @default(dbgenerated("gen_random_uuid()"))
+  rateBy    String // Link to User model
+  clubId    Int // Link to Club model
+  rating    Int           @default(0) // 0 as initial rate lol
+  // ratedAt   DateTime      @default(now()) // I guess we don't need this as we already have `rateScope`
```

Comment:
we should still add it imo. we might need it if we add afterwards some more functions